test/jdk/jdk/jfr/event/oldobject/TestClassLoaderLeak.java

```diff
@@ -39,9 +39,8 @@
 /**
  * @test
  * @requires vm.flagless
- * @requires vm.hasJFR
- * @requires vm.flagless
```

Comment:
Why `vm.flagless` was removed? It stays the same in other tests.

Reply:
There's an extra one on line 41, which remains.

In general, I realize there isn't a canonical order for these, which if it were adhered to would be good.